ssi-script.md

```diff
@@ -0,0 +1,122 @@
+# Script for SSI Fellowship Application
+
+Slides can be found here :point_right: <https://www.dropbox.com/s/js4uoy8stymc6bo/SGibson_SSI_2020_Fellowship_Application.pdf?dl=0>
```

Comment:
I'd suggest putting your slides on Figshare or Zenodo, where you will get a DOI for them and then use the official [short DOI service](http://shortdoi.org/) to get a DOI link short enough to put on the footer of each slide.

Reply:
Assuming you're happy for your slides to be be public that is. It's a pre-recorded video presentation too, right? You could also host the video along with the slides on Figshare/Zenodo if you are comfortable doing so.

Reply:
Yeah, I've basically just done this to get feedback.